distribution/lib/Standard/Base/0.0.0-dev/src/Network/Uri.enso

```diff
@@ -23,7 +23,7 @@ polyglot java import java.util.Optional
 parse : Text -> Uri ! Syntax_Error
 parse text =
     Panic.catch_java Any (Uri (Java_URI.create text)) java_exception->
-        Error.throw (Syntax_Error java_exception.getMessage)
+        Error.throw (Syntax_Error ("Uri syntax error: " + java_exception.getMessage))
```

Comment:
I'm wondering if we shouldn't rename this file to `URI.enso` according with the new guidelines?

Even if not as far as code is concerned, I'd consider if the error message should use the 'URI' form.